server/form-pages/taskListPage.ts

```diff
@@ -13,6 +13,8 @@ export interface TaskListPageInterface {
 }
 
 export default abstract class TaskListPage {
+  abstract documentTitle: string
```

Comment:
I'm thinking this could be better named to show what it intends... `titleWithoutPersonalInformation` ??